index.js

```diff
@@ -321,4 +372,56 @@ input DeleteOtherAnswerInput {
   parentAnswerId: ID!
 }
 
+input CreateRoutingRuleSetInput {
+  questionPageId: ID!
+}
+
+input UpdateRoutingRuleSetInput {
+  id: ID!
+  questionPageId: ID!
```

Comment:
What would this be for?

Reply:
Yeah Sam G raised this question as well. My thinking behind this was that it would be possible to associate routing rule set with another page. So you could disassociate routing with one page and add it to another.

But I'm not sure that's clear from the API and perhaps should be done as a separate mutation.

Do you think we'd be better removing the update mutation for a rule set, or would it be better to change it so that you could change some other stuff? e.g. change the else?

Reply:
i can't imagine there'll be much disassociating going on. i think changing the `else` etc is what you want this for